src/components/Dashboard/Tiles/Widgets/WidgetsEditModal.tsx

```diff
@@ -262,28 +370,58 @@ const WidgetOptionTypeSwitch: FC<{
       );
     case 'multiple-text':
       return (
-        <MultiSelect
-          data={value.map((name: any) => ({ value: name, label: name }))}
-          label={t(`descriptor.settings.${key}.label`)}
-          description={t(`descriptor.settings.${key}.description`)}
-          defaultValue={value as string[]}
-          withinPortal
-          searchable
-          creatable
-          getCreateLabel={(query) => t('common:createItem', { item: query })}
-          onChange={(values) =>
-            handleChange(
-              key,
-              values.map((item: string) => item)
-            )
-          }
-        />
+        <Stack spacing={0}>
+          <Group align="center" spacing="sm">
+            <Text size="0.875rem" weight="500">{t(`descriptor.settings.${key}.label`)}</Text>
+            {info &&
+              <Tooltip.Floating
+                label={t(`descriptor.settings.${key}.info`)}
+                position="right-start"
+                c={ colorScheme === 'light' ? "black" : "dark.0" }
+                color={ colorScheme === 'light' ? "dark.0" : "dark.6" }
+                styles={{ tooltip: { maxWidth: 300, }, }}
+                multiline
+              >
+                <IconAlertCircle size="1.25rem" style={{ display: 'block', opacity: 0.5 }} />
+              </Tooltip.Floating>
```

Comment:
I've never seen the Tooltip.Floating components before. Can't you do <Tooltip floating/> as an attribute ? Just curious

Reply:
it's to have the tooltip follow the cursor instead of being linked to the element. Since it's not the same type of tooltip (no arrow pointing at element) it's a different tooltip I guess. I just followed the docs of tooltip